app/javascript/components/formSubmit.js

```diff
@@ -0,0 +1,15 @@
+export const autoSubmit = () => {
+    const checkBoxes = document.querySelectorAll(".form-check-input");
+    const select = document.querySelector("#search_prefecture")
+    const form = document.querySelector('form');
+    
```

Comment:
```suggestion
    if (!!select)
```
I am getting the same error than last night (error in console and navbar not working) But if i add `if (!!select)` it works!